src/tests/page_objects/planner/planner.page.ts

```diff
@@ -49,4 +49,8 @@ export class PlannerPage extends AppPage {
     await $('body').sendKeys(protractor.Key.ESCAPE);
     await this.quickPreview.notificationToast.untilHidden();
   }
+
+  async waitUntilUrlContains(text: string) {
```

Comment:
Can we move this to `basePage.ts` ?
This is a generic function and should be in base page class.

Reply:
done @jarifibrahim